witchcraft/loggers.go

```diff
@@ -63,7 +64,8 @@ func (s *Server) initLoggers(useConsoleLog bool, logLevel wlog.LogLevel, registr
 	logWriterFn := func(slsFilename string) io.Writer {
 		internalWriter := newDefaultLogOutputWriter(slsFilename, useConsoleLog, loggerStdoutWriter)
 		if tcpWriter != nil {
-			internalWriter = io.MultiWriter(internalWriter, tcpWriter)
+			asyncTCPWriter := tcpjson.StartAsyncWriter(tcpWriter, registry)
```

Comment:
It doesn't look like this writer is ever closed -- what's the thinking behind how the lifecycle should be managed here?

I think this happens to work out right now because, although `initLoggers` is called multiple times on initialization, `tcpWriter` is only non-nil on one of these invocations. However, if it's the case that calling `initLoggers` with a non-nil `tcpWriter` will always start a new goroutine with no way to close/stop it, that seems like a bit of a concern.